modules/exploits/windows/http/sharepoint_ssi_viewstate.rb

```diff
@@ -175,6 +183,9 @@ def create_ssi_page
 
     res = send_request_cgi(
       'method' => 'PUT',
+      'headers' => {
+        'Cookie' => cookie
+      },
       'uri' => ssi_path,
       'data' => ssi_page
```

Comment:
```suggestion
      'method' => 'PUT',
      'uri' => ssi_path,
      'cookie' => cookie,
      'data' => ssi_page
```

This is the order I prefer as well. It's somewhat like the raw HTTP request.